src/WarcraftLegacies.Source/Quests/Frostwolf/QuestSeaWitch.cs

```diff
@@ -29,13 +30,15 @@ public sealed class QuestSeaWitch : QuestData
       AddObjective(new ObjectiveKillUnit(LegendNeutral.LegendSeawitch.Unit));
       AddObjective(new ObjectiveExpire(600));
       ResearchId = FourCC("R05H");
-
-      foreach (var unit in new GroupWrapper().EnumUnitsInRect(rescueRect).EmptyToList())
-        if (GetOwningPlayer(unit) == Player(PLAYER_NEUTRAL_PASSIVE))
+      _rescueUnits = rescueRect.PrepareUnitsForRescue(Player(PLAYER_NEUTRAL_PASSIVE));
+      CreateTrigger()
```

Comment:
I think you meant to assign this to `_rescueTrigger`? Brilliant way to do it, anyway.